modules/ROOT/pages/dw-multipart-functions-form.adoc

```diff
@@ -43,6 +103,8 @@ var secondPart = "content for my second part"
 }
 ----
 
+In this structure, the part keys `part1` and `part2` are stripped out in the `multipart/form-data` output.
+
```

Comment:
DW Source section grammar doesn't allow free text so this description should be Example section or in the Output section to be regenerated from DW's docs.